### PR TITLE
docs: add subject and audit files

### PR DESCRIPTION
**Pull request by zanninso**

### CODE REVIEW

File: subjects/java/Buy01/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+### Instructions
+
+#### 1. Microservices Setup
+
+- Set up your microservice architecture. Consider **Spring Cloud Eureka** for service discovery and **Docker** for containerization.
+
+#### 2. Enhanced Database Design
+
+```mermaid
```
Comment (akhossanX) [inline subjects/java/Buy01/README.md:15]: mermaid not supported by the platform, use ASCII art

File: subjects/java/Buy01/audit/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#### Initial Setup & Access
+
+##### Download the project and initiate it. Try accessing the web pages and the API endpoints using a web browser and tools like Postman.
```
Comment (akhossanX) [inline subjects/java/Buy01/audit/README.md:3]: Maybe better to tell the auditee to run the web app in a docker container

File: subjects/java/Buy01/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+classDiagram
+    User "1" -- "n" Product : Sells
+    Product "1" -- "n" Media : Contains
+    User : +String id
+    User : +String name
+    User : +String email
+    User : +String password
+    User : +String role (client/seller)
```
Comment (akhossanX) [inline subjects/java/Buy01/README.md:23]: consider using Enum ?

**Review by akhossanX — CHANGES_REQUESTED** [review]
all seems to be good, make the appropriate changes then resolve the converstions

File: subjects/java/Buy01/README.md (hunk 3 above)
Comment (zanninso) [reply subjects/java/Buy01/README.md]: done

File: subjects/java/Buy01/audit/README.md (hunk 2 above)
Comment (zanninso) [reply subjects/java/Buy01/audit/README.md]: done

File: subjects/java/Buy01/README.md (hunk 1 above)
Comment (zanninso) [reply subjects/java/Buy01/README.md]: done

**Review by akhossanX — APPROVED** [review]
Seems good

**Review by davhojt — CHANGES_REQUESTED** [review]
Path should be `subjects/java/projects/buy-01`

**Pull request merged**